201-managed-application-usingmsi-for-management/artifacts/ManagedAppZip/mainTemplate.json

```diff
@@ -0,0 +1,175 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().location]"
+        },
+        "logicAppName": {
+            "type": "string",
+            "defaultValue": "logicappwf"
+        },
+        "rolenameguid": {
+            "type": "string"
+        },
+        "storageAccountName": {
+            "type": "string",
+            "defaultValue": "suhas"
+        }
+    },
+    "variables": {
+        "customResourceProviderName": "public",
+        "storageAccountType": "Standard_LRS",
+        "_deploymentLink": "[deployment().properties.templateLink.uri]",
+        "storageBlobOwnerId": "[concat('/subscriptions/', subscription().subscriptionId, '/providers/Microsoft.Authorization/roleDefinitions/', 'b7e6dc6d-f1e8-4753-8033-0f276bb0955b')]"
```

Comment:
remove concat and use subscriptionResourceId() to create the resourceId

Reply:
Done